exes/mqtt-bridge/inc/client.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <cstdint>
+#include <functional>
+#include <optional>
+#include <string>
+#include <variant>
+
+#include <tfc/confman.hpp>
+#include <tfc/logger.hpp>
+
+#include <config/mqtt_config.hpp>
+#include "endpoint.hpp"
```

Comment:
How about forward declaring this entity? That would remove the async_mqtt headers from all but `client.cpp`